WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -370,7 +370,7 @@ private extension ZendeskUtils {
         zendeskIdentity.email = userEmail
         zendeskIdentity.name = ZendeskUtils.sharedInstance.userName
         ZDKConfig.instance().userIdentity = zendeskIdentity
-        DDLogDebug("Zendesk identity created with email '\(zendeskIdentity.email)' and name '\(zendeskIdentity.name)'.")
+        DDLogDebug("Zendesk identity created with email '\(zendeskIdentity.email ?? "")' and name '\(zendeskIdentity.name ?? "")'.")
```

Comment:
Sorry to be nitpicky – I think the issue here was not that the email / name might have no value, but rather even if they have a value they're printed as `Optional(demo@example.com)`.

Reply:
With this change, the log message now looks like:
`Zendesk identity created with email 'scoutharrissites+2@gmail.com' and name ''.`

Reply:
You're right, it looks fine to me! Sorry!

Reply:
No prob. Thanks!